contracts/liquidityMining/KyberFairLaunchV2.sol

```diff
@@ -133,11 +134,15 @@ contract KyberFairLaunchV2 is IKyberFairLaunchV2, PermissionAdmin, ReentrancyGua
   ) PermissionAdmin(_admin) {
     rewardTokens = _rewardTokens;
     rewardLocker = _rewardLocker;
-
     // approve allowance to reward locker
+    multipliers = new uint256[](_rewardTokens.length);
     for (uint256 i = 0; i < _rewardTokens.length; i++) {
```

Comment:
I learnt recently it's cheaper to cache array length.
```
uint256 length = _rewardTokens.length;
for (uint256 i = 0; i < _rewardTokens.length; i++) {
...
```
small gas saving... but not that important